pkg/testutil/testutil.go

```diff
@@ -41,3 +42,29 @@ func CompileInTempDir(t testing.TB) (tmpDir string, execPath string) {
 	}
 	return
 }
+
+// ErrorExists is only used to increase readability of future tests
```

Comment:
this is a nice idea but ... it's not how people write Go. I recommend just sticking with the common pattern of 
err != nil

Reply:
Fixed